src/components/scenes/MigrateWalletCompletionScene.tsx

```diff
@@ -239,12 +239,11 @@ const MigrateWalletCompletionComponent = (props: Props) => {
 
   return (
     <SceneWrapper background="theme">
-      {({ safeAreaInsets }) => (
-        <View style={[styles.content, { marginBottom: -safeAreaInsets.bottom }]}>
+      {({ insetStyles }) => (
+        <View style={[styles.content, insetStyles]}>
```

Comment:
Same here. We want the list to go underneath the tab bar.

Reply:
Same issue here. We're using a secondary button instead of a primary so we don't have a floating button